### PR TITLE
Add features list to README.md

### PR DESCRIPTION
**Pull request by jakelangham**
This is to advertise the features of the software better and is also a requirement of our ongoing JOSS submission.

### CODE REVIEW

**Review by markwoodhouse — APPROVED** [review]
This looks good.

Very good to keep this concise and good to have the links to more info.

**Comment by jakelangham** [comment]
Anything missing do you think, or shall I just merge and move on to the next thing?

**Comment by markwoodhouse** [comment]
Two other features could be:
- Eddy-diffusivity to ensure well-posedness of the coupled hydrodynamics and morphodynamics.
- Variety of boundary conditions for initiating flows, including from multiple locations.

Happy to consider more if it is better to be thorough now.

**Comment by jakelangham** [comment]
I've added those points - see what you think. Maybe the eddy viscosity one is a bit pedantic and verbose? I'm minded to take it out...

File: README.md [hunk 1]
```diff
@@ -3,6 +3,27 @@
...
+
+The software is designed to be easy to install and use. Simulations approximate
+solutions to a underlying set of shallow-layer equations that describe a generic
+flowing mixture of fluid and sediment. A modular design allows a variety of popular 
+closures to be selected at runtime to specialise simulations for particular purposes.
+
+* Fully conservative well-balanced positivity-preserving finite volume solver.
+* Dynamic erosion and deposition of sediment.
+* Eddy viscosity implementation to ensure well-posedness of morphodynamic flows
+  (see [here](https://arxiv.org/abs/2007.15989) and [here](https://arxiv.org/abs/2306.16185) for detailed discussions of this issue).
```
Comment (markwoodhouse) [inline README.md:16]: I'd suggest moving the citations to go with the other references.

File: README.md [hunk 2]
```diff
@@ -3,6 +3,27 @@
...
+* User-settable basal drag and morphodynamics parametrisations.
+* Simulation on user-specified topographies: simple geometric surfaces and digital elevation maps (DEMs).
+* Variety of boundary conditions for initiating flows, including from multiple locations.
+* Output via geo-referenced NetCDF or plain-text.
+
+An overview of the physical modelling framework may be found 
+[here](https://kestrel-unibristol.readthedocs.io/en/latest/physical%20model.html).
+For full details, including the numerical implementation, consult our 
+[paper](https://arxiv.org/abs/2306.16185).
```
Comment (markwoodhouse) [inline README.md:25]: Here could add:
"Eddy viscosity ensures the morphodynamic system is well-posed.  see [here](https://arxiv.org/abs/2007.15989) and [here](https://arxiv.org/abs/2306.16185) for detailed discussions of this issue."

**Review by markwoodhouse — CHANGES_REQUESTED** [review]
Suggest moving citations from bullet-point list

**Comment by jakelangham** [comment]
I think the last commit should resolve this? See what you think.

**Review by markwoodhouse — APPROVED** [review]
This is good.  Happy to finalize this.

**Pull request merged**